cypher/cypher-docs/src/docs/dev/ql/functions/index.asciidoc

```diff
@@ -151,18 +151,18 @@ All trigonometric functions operate on radians, unless otherwise specified.
 [options="header"]
 |===
 | Function                           | Description
-| <<functions-acos, acos()>>         | Returns the arccosine of the expression.
-| <<functions-asin, asin()>>         | Returns the arcsine of the expression.
-| <<functions-atan, atan()>>         | Returns the arctangent of the expression.
-| <<functions-atan2, atan2()>>       | Returns the arctangent2 of a set of coordinates.
-| <<functions-cos, cos()>>           | Returns the cosine of the expression.
-| <<functions-cot, cot()>>           | Returns the cotangent of the expression.
+| <<functions-acos, acos()>>         | Returns the arccosine of a number in radians.
```

Comment:
I think I've commented elsewhere about my concerns regarding the removal of the term 'expression'. These comments apply here to all these changes too.